libs/ui/package.json

```diff
@@ -74,6 +74,7 @@
     "react-modal": "^3.16.1",
     "react-router-dom": "^5.3.4",
     "react-select": "^5.7.4",
+    "sanitize-html": "^2.13.0",
```

Comment:
Why introduce another HTML sanitizer when we already have `dompurify`? If this one is better in a meaningful way, should we switch all uses of `dompurify` over to this? I'd like to avoid having both if possible.

Reply:
Great question - `dompurify` only runs in the browser. There is a workaround where you can use `jsdom` to make it run in Node, but that felt a bit heavy, so I looked for slimmer alternatives.

I'm definitely open to consolidating - maybe will do a follow up PR to consolidate. But first I want to make sure we agree on consolidating. Do you think that's a good path, or do you have other thoughts?

Reply:
They're both quite popular on NPM (~2M for `sanitize-html` and ~6M for `dompurify` weekly downloads). I thought `sanitize-html` seemed like the newer maybe-successor to `dompurify`, but it's actually older. It does seem to be more configurable and is meant more for running in NodeJS, so I think I'm okay consolidating on it over `dompurify`.

Reply:
https://github.com/votingworks/vxsuite/pull/5392